package.json

```diff
@@ -97,6 +97,7 @@
     "node-sass": "^3.4.2",
     "object-map-to-array": "^1.0.0",
     "onionsay": "github:theonion/onionsay",
+    "pdfobject": "pipwerks/pdfobject#d37c492c026dc8abeac636d1d5d6eeebdb409e8b",
```

Comment:
Might be worth a comment here explaining this is intentionally pointing at a hash for use in production.